src/FsAutoComplete/LspServers/AdaptiveServerState.fs

```diff
@@ -2051,26 +2129,45 @@ type AdaptiveState(lspClient: FSharpLspClient, sourceTextFactory: ISourceTextFac
             && file.Contains "AssemblyAttributes.fs" |> not)
 
         let checksToPerformLength = innerChecks.Length
+        let rootToken = sourceFilePath |> getOpenFileTokenOrDefault
 
         innerChecks
-        |> Array.map (fun (proj, file) ->
-          let file = UMX.tag file
+        |> Array.map (fun (opts, file) ->
```

Comment:
These are changes ported from the `nightly` branch since I made them in the transparent compiler.